src/dotty/tools/dotc/reporting/Reporter.scala

```diff
@@ -12,44 +12,9 @@ import config.Printers
 import java.lang.System.currentTimeMillis
 import typer.ErrorReporting.DiagnosticString
```

Comment:
Good analysis. Can you take it over and integrate the PR request directly
into your work?

On Wed, Feb 17, 2016 at 6:53 PM, Guillaume Martres <notifications@github.com
> wrote:

> In src/dotty/tools/dotc/reporting/Reporter.scala
> <https://github.com/lampepfl/dotty/pull/1100#discussion_r53202952>:
>
> > @@ -12,44 +12,9 @@ import config.Printers
> >  import java.lang.System.currentTimeMillis
> >  import typer.ErrorReporting.DiagnosticString
>
> I think the weird test errors are because DiagnosticString does not exist
> anymore.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lampepfl/dotty/pull/1100/files#r53202952>.
>



-- 
Martin Odersky
EPFL